### ISSUE TITLE
Synthetic Dataset Train/Test Split

### ISSUE THREAD

**Issue by Saumya-Gupta-26** [giesekow/deepvesselnet#16]
Hello,

Regarding your synthetic dataset, from your paper it is written, "_Twenty volumes out of the 136 is used as a test set and the remaining volumes are used for pretraining our networks in the various tasks at hand_".

I was wondering if you could mention which volumes were used for training and which were used for testing? I would try the same experimental setup then.

Thank you so much!
